README.md

```diff
@@ -209,6 +210,18 @@ If you have access to [Red Hat Catalog](https://access.redhat.com/containers/#/r
 
 [In future versions](https://github.com/m88i/nexus-operator/issues/14) the Operator will handle this step for you.
 
+## Image Pull Policy
+
+You can control the pods Image Pull Policy using the `spec.imagePullPolicy` field. It accepts either of the following values:
+
+  - `Always`
+  - `IfNotPresent`
+  - `Never` 
+
+If this field is set to an invalid value this configuration will be omitted, deferring the behavior to Kubernetes default, which is `Always` if the image's tag is "latest" and `IfNotPresent` otherwise.
```

Comment:
Do we have a reference for this statement (like a Kube doc)? I remember @Kaitou786 brought this up, maybe he has it? :)

Reply:
@ricardozanini From K8s all I found was [this](https://kubernetes.io/docs/concepts/containers/images/#updating-images), which kind of breezes over the matter, but I think is clear enough.

On the other hand, [Openshift's doc](https://docs.openshift.com/container-platform/3.6/dev_guide/managing_images.html#image-pull-policy) gives it more attention, with its own section (though the conclusions we can draw from both are ultimately the same, of course).

I suppose we can link the K8s one, unless @Kaitou786 actually saw this somewhere else.

Reply:
+1 to add the Kuberentes ref. :)

Reply:
The kubernetes ref seems better, origanlly I read it in comments in the `Container Struct` where this is defined

Reply:
@LCaparelli as soon as you update the docs, we can merge this.